common4j/src/main/com/microsoft/identity/common/java/util/ported/ObjectUtils.java

```diff
@@ -48,4 +49,13 @@ public static boolean equals(@Nullable final Object o1, @Nullable final Object o
         return (o1 == o2) || (o1 != null && o1.equals(o2));
     }
 
+    /**
+     * A helper function for validating if the given object is null or empty.
+     */
+    public static void throwIfObjectIsNull(@NonNull final String objectName, @Nullable final Object object) throws IllegalStateException {
+        if (object == null) {
+            throw new IllegalStateException(objectName + " is null or empty.");
```

Comment:
You may want to throw a `NullPointerException` instead

Reply:
7cee1dd4643a39521911892cd529e5ca2bfae6fb